sitewit/models.py

```diff
@@ -0,0 +1,54 @@
+from sitewit.services import SitewitService
+
+
+class User(object):
+    def __init__(self, name, email, token):
+        self.name = name
+        self.email = email
+        self.token = token
+
+
+class Account(object):
+    _sitewitservice = None
+
+    def __init__(self, account_data, user_data=None):
+        self.id = account_data['accountNumber']
+        self.token = account_data['token']
+        self.status = account_data['status']
+        self.url = account_data['url']
+        self.site_id = account_data['clientId']
+        self.currency = account_data['currency']
+        self.country_code = account_data['country']
+
+        if user_data is not None:
+            self.user = User(user_data['name'], user_data['email'],
+                             user_data['token'])
+
+    @classmethod
+    def get_service(cls):
+        if cls._sitewitservice is None:
+            cls._sitewitservice = SitewitService()
+        return cls._sitewitservice
+
+    @classmethod
+    def create(cls, user, site, url, user_token=None):
```

Comment:
We can't do this, because SiteWit requires URL with prefix when account is updated or created.